pyproject.toml

```diff
@@ -34,67 +30,99 @@ classifiers = [
     "Topic :: Utilities",
     "Typing :: Typed",
 ]
-packages = [{ include = "polyfactory" }]
-
-[tool.poetry.dependencies]
-python = ">=3.8,<4.0"
-beanie = { version = "*", optional = true }
-faker = "*"
-msgspec = { version = "*", optional = true }
-odmantic = { version = "*", optional = true }
-pydantic = { version = "*", optional = true, extras = ["email"] }
-typing-extensions = "*"
-attrs = {version = ">=22.2.0", optional = true}
-sqlalchemy = { version = ">=2", optional = true }
-
-[tool.poetry.group.dev.dependencies]
-hypothesis = "*"
-mongomock-motor = "*"
-pytest = "*"
-pytest-asyncio = "*"
-pytest-cov = "*"
-aiosqlite = "^0.19.0"
-
-[tool.poetry.group.docs.dependencies]
-auto-pytabs = "*"
-blacken-docs = "*"
-litestar-sphinx-theme = { git = "https://github.com/litestar-org/litestar-sphinx-theme.git" }
-sphinx = "*"
-sphinx-autobuild = "*"
-sphinx-copybutton = "*"
-sphinxcontrib-mermaid = "*"
-
-[tool.poetry.group.lint]
-optional = true
-
-[tool.poetry.group.lint.dependencies]
-black = "*"
-blacken-docs = "*"
-mypy = "*"
-pre-commit = "*"
-pyright = "*"
-ruff = '*'
-sourcery = "*"
+name = "polyfactory"
+version = "2.9.0"
+description = "Mock data generation factories"
+readme = "docs/PYPI_README.md"
+license = {text = "MIT"}
+requires-python = ">=3.8,<4.0"
+dependencies = [
+    "faker>=19.6.2",
+    "typing-extensions>=4.8.0",
+]
 
-[tool.poetry.extras]
-sqlalchemy = ["sqlalchemy"]
+[project.optional-dependencies]
```

Comment:
I am not certain on this section.

Reply:
Did you mean the version constraints? Currently, they seem to be incorrect. We only have constraints for `sqlalchemy` and `attrs` for versions `2.0.0` and `22.2.0` respectively.